src/main/java/org/folio/rest/client/CirculationStorageClient.java

```diff
@@ -29,4 +42,28 @@ public Future<LoanPolicy> getLoanPolicyById(String id) {
   public Future<Collection<LoanPolicy>> getLoanPoliciesByIds(Collection<String> ids) {
     return getByIds("/loan-policy-storage/loan-policies", ids, LoanPolicy.class, "loanPolicies");
   }
+
+  public Future<ActualCostRecord> fetchActualCostRecordById(String actualCostRecordId) {
+    return getById(ACTUAL_COST_RECORD_STORAGE_URL, actualCostRecordId, ActualCostRecord.class);
+  }
+
+  public Future<ActualCostRecord> updateActualCostRecord(ActualCostRecord actualCostRecord) {
+    Promise<HttpResponse<Buffer>> promise = Promise.promise();
+    okapiPutAbs(ACTUAL_COST_RECORD_STORAGE_URL, actualCostRecord.getId())
+      .sendJson(actualCostRecord, promise);
+
+    return promise.future().compose(response -> {
```

Comment:
No need for a Promise:
```suggestion
    return okapiPutAbs(ACTUAL_COST_RECORD_STORAGE_URL, actualCostRecord.getId())
      .sendJson(actualCostRecord)
      .compose(response -> {
```